install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -128,6 +128,7 @@ type WorkspaceConfig struct {
 	} `json:"contentService"`
 
 	EnableProtectedSecrets *bool `json:"enableProtectedSecrets"`
+	UseWsmanagerMk2        bool  `json:"useWsmanagerMk2,omitempty"`
```

Comment:
I would strongly advocate not putting into the `experimental` section as I'm currently in the process of deprecating as much as possible from here.

The only exception would be if this is a truly experimental feature that is only going to be enabled in a limited fashion **AND** a feature that we're unlikely to promote to LTS - for example, if the intention is to use ws-manager2 once we've gotten through testing. If the plan is to support both ws-manager 1 and 2 concurrently, this should be in the main config.

Reply:
@mrsimonemms This property will be removed once we have rolled out mk2 to production. Mk1 will be deleted from the repository and mk2 will be the only relevant workspace manager going forward. See [milestone plan](https://www.notion.so/gitpod/Milestones-ws-manager-mk2-fb471d7e01e4494b9bca9ad1cd50b3b2).

Reply:
Perfecto!